.changeset/tough-planets-reply.md

```diff
@@ -0,0 +1,12 @@
+---
+'@primer/react-brand': patch
+---
+
+Add `0` option to Box spacing props to reset defined paddings at larger viewports.
```

Comment:
What about using `none` instead? 

I think we use that to reset defined spaces in Primer Web components: https://primer.style/design/components/page-layout/react#pagelayout

Reply:
That's a good shout 👍. We have precedent for this in PB already, as Hero and a few other components already have 'none' options. Not sure how I missed that 😅